stable/sumologic-fluentd/templates/daemonset.yaml

```diff
@@ -227,18 +227,18 @@ spec:
           configMap:
             name: {{ template "sumologic-fluentd.fluentdUserConfig.fullname" . }}
         {{- end }}
-    {{- if .Values.tolerations }}
+{{- if .Values.tolerations }}
       tolerations:
-    {{ toYaml .Values.tolerations | indent 8 }}
-    {{- end }}
-    {{- if .Values.nodeSelector }}
+{{ toYaml .Values.tolerations | indent 8 }}
+{{- end }}
+{{- if .Values.nodeSelector }}
       nodeSelector:
-    {{ toYaml .Values.nodeSelector | indent 8 }}
-    {{- end }}
-    {{ if .Values.affinity }}
+{{ toYaml .Values.nodeSelector | indent 8 }}
+{{- end }}
+{{ if .Values.affinity }}
       affinity:
-    {{ toYaml .Values.affinity | indent 8 }}
-    {{- end }}
+{{ toYaml .Values.affinity | indent 8 }}
+{{- end }}
```

Comment:
I think this should stay indented.

Reply:
(tracking in a single conversation)

Reply:
wait no, my eyes were wrong, different segments

Reply:
The `{{ toYaml }}` blocks need to be un-indented like similar blocks above, otherwise the first line is "out of sync". It might be safe to indent if we change it to `{{- toYaml }}`, I'm not completely clear on the syntax there.

The other instances of indentation I can change.